src/components/uploadPicture/uploadImage.tsx

```diff
@@ -111,27 +113,23 @@ const UploadImageComponent: React.FC<UploadImageComponentProps> = ({
   }, [uploadSuccess]);
 
   return (
-    <div className={styles.uploadContainer}>
+    <div className={styles.uploadContainer} onClick={onClickStartButton}>
```

Comment:
Anything that functions as a button should probably use `button` tags. But should this onclick be on the container? Shouldn't it be on the div on line 118?

Reply:
will do. thanks for the feedback!